ga4gh/server/backend.py

```diff
@@ -414,16 +414,25 @@ def resultsHandler(self, results, patient_list, dataset_id, return_mimetype, acc
         # }
 
         table = results[0].get("table")
-        field = results[0].get("field")
+        field = results[0].get("fields")
```

Comment:
This is related to the change in complex query - the variable name is however, unchanged.